forge-gui/res/cardsfolder/upcoming/sister_of_silence.txt

```diff
@@ -0,0 +1,8 @@
+Name:Sister of Silence
+ManaCost:4 U
+Types:Creature Human Knight
+PT:3/3
+K:Flash
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigCounter | TriggerDescription$ Psychic Abomination — When CARDNAME enters the battlefield, counter target instant spell, sorcery spell, activated ability, or triggered ability.
+SVar:TrigCounter:DB$ Counter | TargetType$ Spell,Activated,Triggered | TgtPrompt$ Select target instant spell, sorcery spell, activated ability, or triggered ability | ValidTgts$ Instant,Sorcery,Emblem
```

Comment:
This ValidTgts seems wrong to me... what about activated abilities of permanents?